changelog.d/5-internal/in-mem-specs

```diff
@@ -0,0 +1 @@
+Added laws for DefaultSsoCode, Now, IdP and ScimExternalIdStore
```

Comment:
this may be a bit confusing for the casual reader of the release notes.  i'll fix it on develop after merging.